artifacts/definitions/Windows/EventLogs/Kerbroasting.yaml

```diff
@@ -0,0 +1,71 @@
+name: Windows.EventLogs.Kerbroasting
+description: |
+
+  This Artifact will return all successful Kerberos TGS Ticket events for Service Accounts (SPN attribute) implemented with weak encryption. These tickets are vulnerable to brute force attack and this event is an indicator of a Kerbroasting attack.  
+  
+  Typical attacker methodology is to firstly request accounts in the domain with SPN attributes, then request an insecure TGS ticket for brute forcing. This attack is particularly effective as any domain credentials can be used to implement the attack and service accounts often have elevated privileges. Kerbroasting can be used for privilege escalation or persistence by adding a SPN attribute to an unexpected account.  
+  
+  **ATT&CK**: [T1208 Kerbroasting](https://attack.mitre.org/techniques/T1208/)  
+  **Reference**: [The Art of Detecting Kerberoast Attacks](https://www.trustedsec.com/2018/05/art_of_kerberoast/)  
+  **Log Source**: Windows Security Event Log (Domain Controllers).  
+  **Event ID**: 4769.  
+  **Status**: 0x0 (Audit Success).  
+  **Ticket Encryption**: 0x17 (RC4).  
+  **Service Name**: NOT krbtgt or NOT a system account (account name ends in $).  
+  **TargetUserName**: NOT a system account (*$@*).  
+  
+  
+  Monitor and alert on unusual events with these conditions from an unexpected IP.  
+  Note: There are potential false positives so whitelist normal source IPs and manage risk of insecure ticket generation.
+
+
+author: Matt Green - @mgreen27
+
+parameters:
+  - name: eventLog
+    default: C:\Windows\system32\winevt\logs\Security.evtx
+  - name: accessor
+    default: ntfs
```

Comment:
You no longer need to explicitly specify ntfs because it will just drop to ntfs on demand

Reply:
removed